src/data/featured_projects.json

```
@@ -61,5 +47,19 @@
     "repo": "osquery-go",
     "star_count": 190,
     "url": "https://github.com/kolide/osquery-go"
```

```suggestion
    "url": "https://github.com/osquery/osquery-go"
```

As long as you're here...

---

Fixed it